src/ui/include/docengine.h

```diff
@@ -74,7 +75,7 @@ class DocEngine : public QObject
     bool read(QFile *file, Editor *editor);
     bool read(QFile *file, Editor *editor, QTextCodec *codec, bool bom);
     // FIXME Separate from reload
-    bool loadDocuments(const QList<QUrl> &fileNames, EditorTabWidget *tabWidget, const bool reload, QTextCodec *codec, bool bom);
+    bool loadDocuments(const QList<QUrl> &fileNames, EditorTabWidget *tabWidget, const bool reload, QTextCodec *codec, bool bom, bool rememberLastSelectedDir=true);
```

Comment:
Need a comment block to explain what `rememberLastSelectedDir` does.

Don't worry about documenting the other parameters, I'll need to clean my own mess some day.